plugin/system_copy.vim

```diff
@@ -6,7 +6,7 @@ let s:mac = 'mac'
 let s:windows = 'windows'
 let s:linux = 'linux'
 
-if exists('g:loaded_system_copy') || &cp || v:version < 700
+if exists('g:loaded_system_copy') || v:version < 700
```

Comment:
From :h I found that it returns count of previous errors in file, so while there are only definitions I think we can omit this check.